src/handlebars/support.handlebars

```diff
@@ -59,6 +59,360 @@
         and we will work with you and the appropriate development team to resolve it.
         Any fixes we identify will be delivered as part of the next AdoptOpenJDK release.
       </p>
+	  
+		<h3 id="openjdk8_platforms">OpenJDK 8</h3>
+		<p>Where available, OpenJDK 8 binaries are supported on the minimum operating system levels shown in the following tables:</p>
+
+		<table border="1">
+			<thead>
+				<tr>
+					<th>Key</th>
+					<th>Value</th>
+				</tr>
+			</thead>
+			<tbody>
+				<tr>
+					<td>HotSpot variant</td>
+					<td><i class="fa fa-star-o" style="color:orange" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span></td>
+				</tr>
+				<tr>
+					<td>OpenJ9 variant</td>
+					<td><i class="fa fa-star-o" style="color:purple" aria-hidden="true"></i><span class="sr-only">OpenJ9 VM</span></td>
+				</tr>
+			</tbody>
+		</table>
+
+		</br>
+
+		<table border="1">
+			<thead>
+				<tr>
+					<th>Linux</th>
+					<th>x64</th>
+					<th>ppc64le</th>	
+					<th>s390x</th>
+					<th>aarch64</th>
```

Comment:
I think we should also add the aarch32/arm32 for OpenJDK 11 as well

Reply:
I added this, but I couldn't see any 32-bit ARM binaries. Are there any for HotSpot?

Reply:
Yes we have some hotspot ones, also OpenJ9 actually - perhaps not hooked up to the front page.